Project/src/MakeCall/CallCaption.js

```diff
@@ -20,7 +20,7 @@ const CallCaption = ({ call }) => {
 
     const startCaptions = async () => {
         try {
-            if (!captions.isCaptionsActive) {
+            if (!captions.isCaptionsActive || !captions.isCaptionsFeatureActive) {
```

Comment:
do we have two properties on the captions object `isCaptionsActive` and `isCaptionsFeatureActive` ? (or does the acs feature property just need updated in a future beta version?)

Reply:
Yes. There are 2 different captions, one for ACS identities (ACS Captions) and the other for Teams Identities (TeamsCaptions)